enso/experiment/__init__.py

```diff
@@ -162,8 +167,32 @@ def _run_sub_experiment(
 
     def _run_experiment(self, dataset_name, current_setting, experiments):
         """Responsible for running all experiments specified in config."""
-        results = pd.DataFrame(columns=self.columns)
         dataset = self._load_dataset(dataset_name, current_setting.get("Featurizer"))
+        if EXPERIMENT_PARAMS:
+            exp_params = deepcopy(EXPERIMENT_PARAMS)
+            # add any experiments that were not included
+            for experiment_cls in experiments:
+                if experiment_cls.__name__ not in exp_params:
+                    exp_params[experiment_cls.__name__] = {}
+            # add the 'All' configuration to all the experiments
+            if 'All' in EXPERIMENT_PARAMS.keys():
```

Comment:
maybe some .lower() or .upper() comparison to make sure someone doesn't get odd behavior when they forget to capitalize

Reply:
i.e. 

```if "ALL" in {k.upper() for k in EXPERIMENT_PARAMS}```

Reply:
:+1:

Reply:
:+1: